apps/to-do-api/src/adapters/azure/cosmosdb/TaskRepository.ts

```diff
@@ -9,6 +9,11 @@ import { decodeFromFeed, decodeFromItem } from "./decode.js";
 import { cosmosErrorToDomainError } from "./errors.js";
 
 export const makeTaskRepository = (container: Container): TaskRepository => ({
+  delete: (id) =>
+    pipe(
+      TE.tryCatch(() => container.item(id).delete(), E.toError),
```

Comment:
do we need a partition key as well in the delete op?

`container.item(id, partitionKey)`